### PR TITLE
[MISP] search for threats in aliases when guess_threats_from_tags is ON

### PR DESCRIPTION
**Pull request by yassine-ouaamou**
### Proposed changes

*
*

### Related issues

* #2424 
*

### Checklist

- [ ] I consider the submitted work as finished
- [ ] I tested the code for its functionality using different use cases
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

File: external-import/misp/src/misp.py [hunk 1]
```diff
@@ -1879,9 +1879,20 @@ def prepare_elements(self, galaxies, tags, author, markings):
                     threats = self.helper.api.stix_domain_object.list(
                         types=["Intrusion-Set", "Malware", "Tool", "Attack-Pattern"],
                         filters={
```
Comment (Archidoit) [inline external-import/misp/src/misp.py:1881]: You can write this filter in a simpler way : 
filters = {
   "mode": "and",
   "filters": [
      {"key": ["name", "xi_mitre_id", "alias"], "values": [tag_value]}
   ],
   "filterGroups": [],
}

File: external-import/misp/src/misp.py [hunk 2]
```diff
@@ -1879,9 +1879,20 @@ def prepare_elements(self, galaxies, tags, author, markings):
...
-                            "mode": "and",
+                            "mode": "or",
                             "filters": [
-                                {"key": ["name", "x_mitre_id"], "values": [tag_value]}
+                                {
```
Comment (Archidoit) [inline external-import/misp/src/misp.py]: You can write this filter in a simpler way : 
filters = {
   "mode": "and",
   "filters": [
      {"key": ["name", "x_mitre_id", "alias"], "values": [tag_value]}
   ],
   "filterGroups": [],
}

Comment (Archidoit) [reply external-import/misp/src/misp.py]: this is equivalent to : name = tag_value OR x_mitre_id = tag_value OR alias = tag_value

Comment (SamuelHassine) [reply external-import/misp/src/misp.py]: {"key": ["name", "x_mitre_id", "aliases", "x_opencti_aliases"], "values": [tag_value]}

**Pull request merged**